src/Main.purs

```diff
@@ -166,7 +165,11 @@ commands :: Array Args.Command
 commands = [
   Args.command "init" "Generate an example PureScript project." Nothing Init.action [
      Args.option "force" ["--force"] Type.flag
-       "Overwrite any project found in the current directory."
+       "Overwrite any project found in the current directory.",
+     Args.option "forceEff" ["--force-eff"] Type.flag
+       "Overwrite the detected compiler version and generate project using Eff.",
```

Comment:
I don't think `force` is quite the right word here. How about `--with-eff` / `--with-effect` instead?

Reply:
Also for the help, I don't think the compiler version is really being overwritten here; how about "Generate the project using Eff, regardless of the detected compiler version" instead?